template.rb

```diff
@@ -18,6 +18,19 @@ def hotwire?
   !options[:skip_hotwire]
 end
 
+@announcements = {}
```

Comment:
I noticed the previous way I was doing announcements wasn't always getting shown at the end of rails new. This is just me correcting that, though might be most useful (or deleted) in #16